### PR TITLE
Feat/kritika/2808

### PR DESCRIPTION
**Pull request by alexfurmenkov**
This PR adds a new operator - is_ordered_by. The code is written by Kritika and Aleksei but Kritika doesn't have access to the repo, so I am creating the PR from my name. @joebenclark coupld you please add the permissions to Kritika (her username - @kriti-prog).

### CODE REVIEW

File: tests/test_operators.py [hunk 1]
```diff
@@ -2028,6 +2028,55 @@ def test_has_same_values(self):
         result = DataframeType({"value": invalid_df, }).has_same_values({"target": "MHCAT", })
         self.assertTrue(result.equals(pandas.Series([True, True, True, True, True, ])))
 
+    def test_is_ordered_by(self):
```
Comment (nhaydel) [inline tests/test_operators.py:2031]: Can we add a test for dates and strings?

File: business_rules/operators.py [hunk 2]
```diff
@@ -923,6 +922,18 @@ def has_different_values(self, other_value: dict):
     def has_same_values(self, other_value: dict):
         return ~self.has_different_values(other_value)
 
+    @type_operator(FIELD_DATAFRAME)
+    def is_ordered_by(self, other_value: dict) -> pd.Series:
```
Comment (nhaydel) [inline business_rules/operators.py:926]: Do we need to have a flag for descending vs. ascending?

Comment (kriti-prog) [reply business_rules/operators.py:926]: Added in the latest commit

File: tests/test_operators.py (hunk 1 above)
Comment (kriti-prog) [reply tests/test_operators.py:2031]: Added in latest commit

**Review by nhaydel — APPROVED** [review]

**Pull request merged**